spec/cc/engine/analyzers/php/main_spec.rb

```diff
@@ -2,12 +2,17 @@
 require 'cc/engine/analyzers/php/main'
 require 'cc/engine/analyzers/reporter'
 require 'cc/engine/analyzers/engine_config'
+require 'cc/engine/analyzers/file_list'
 require 'flay'
 require 'tmpdir'
 
 module CC::Engine::Analyzers::Php
   describe Main do
-    before { @code = Dir.mktmpdir }
+    before do
+      @code = Dir.mktmpdir
+      Dir.chdir(@code)
```

Comment:
For now there's a `minitest/around` library (I think that's the name) that we've used in other places. I'm in favor of anyone that wants to migrate a codebase to RSpec though. I repeatedly headdesk over various aspects of minitest when I find myself in such a codebase.